test/VectorExt/vector_predication.mlir

```diff
@@ -50,3 +50,27 @@ func @tiled_loop_1d(%arg0: tensor<?xf32>,
 // CHECK-NEXT:        }
```

Comment:
`-` instead of `_` in test names (we can also do as a followup on all file names)

Reply:
also should this test filename specifically spell out "util" ?
Since this is not semantically preserving I would just call it "vector-predication".